modules/lvms-upgrading-lvms-on-sno.adoc

```diff
@@ -6,17 +6,61 @@
 [id="lvms-upgrading-lvms-on-sno_{context}"]
 = Upgrading {lvms} on {sno} clusters
 
-Currently, it is not possible to upgrade from {rh-storage} Logical Volume Manager Operator 4.11 to {lvms} 4.12 on {sno} clusters.
+You can upgrade the {lvms-first} Operator to align with your {sno} version.
 
-[IMPORTANT]
-====
-The data will not be preserved during this process.
-====
+// Is data preserved now?
+
+.Prerequisites
+
+* You have upgraded your {sno} cluster.
+
+* You have installed a previous version of the {lvms} Operator.
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in as a user with `cluster-admin` privileges.
 
 .Procedure
 
-. Back up any data that you want to preserve on the persistent volume claims (PVCs).
-. Delete all PVCs provisioned by the {rh-storage} Logical Volume Manager Operator and their pods.
-. Reinstall {lvms} on {product-title} 4.12.
-. Recreate the workloads.
-. Copy the backup data to the PVCs created after upgrading to 4.12.
\ No newline at end of file
+. Update the `Subscription` resource for the {lvms} Operator by running the following command:
++
+[source,terminal]
+----
+$ oc patch subscription lvms-operator -n openshift-storage --type merge --patch '{"spec":{"channel":"<update-channel>"}}' <1>
+----
+<1> Replace `<update-channel>` with the version of the {lvms} Operator that you want to install, for example `stable-{product-version}`.
+
+. View the upgrade events to check the installation is complete by running the following command:
```

Comment:
Rewrite as: "To check that the installation is complete, view the upgrade events by running the following command:"